content/300-guides/200-deployment/201-serverless/400-deploy-to-aws-lambda.mdx

```diff
@@ -1,301 +1,199 @@
+### Deploy only the required files
 
-## 1. Download the example project
+To reduce your deployment footprint, you can change your deployment process so that you only upload the files that your application needs. The Serverless configuration file `serverless.yml` includes a package pattern that includes only the Prisma engine binary relevant to the Lambda runtime, and excludes the others. This means that when Serverless packages your app for upload, it includes only one binary. This ensures the packaged archive is as small as possible.
 
-To start, download the example project. The project includes the configuration that you need to complete this guide.
-
-### Steps
-
-1. Open a terminal window and navigate to a location of your choice. Use the following commands to create a directory for the project and download the example code:
-
-   ```terminal wrap
-   mkdir prisma-aws-lambda
-   cd prisma-aws-lambda
-   curl https://codeload.github.com/prisma/prisma-examples/tar.gz/latest | tar -xz --strip=3 prisma-examples-latest/deployment-platforms/aws-lambda/
-   ```
-
-2. Install the dependencies:
-
-   ```terminal
-   npm install
-   ```
-
-<!-- tar strip folder is a concatenation of the REPOSITORY-BRANCH/REF, e.g. prisma-examples-latest -->
-
-### Result
-
-Your directory contains the following files:
-
-```terminal no-copy
-README.md
-handlers
-node_modules
-package-lock.json
-package.json
-prisma
-schema.sql
-serverless.yml
+```code file=serverless.yml no-copy
+package:
+  patterns:
+    - '!node_modules/.prisma/client/libquery_engine-*'
+    - 'node_modules/.prisma/client/libquery_engine-rhel-*'
+    - '!node_modules/prisma/libquery_engine-*'
+    - '!node_modules/@prisma/engines/**'
 ```
 
-## 2. Set the DATABASE_URL in <inlinecode>.env</inlinecode>
-
-The Lambda functions need the `DATABASE_URL` environment variable to access the database. You must define an `.env` file. The preconfigured `serverless-dotenv-plugin` uses this file to inject into the function runtime.
-
-The repository contains an `.env.example` example file to help with this.
-
-### Steps
-
-1. Enter the following command to copy `.env.example` to a new file, `.env`:
-
-   ```terminal
-   cp .env.example .env
-   ```
-
-2. Open `.env` and set the `DATABASE_URL` to your database credentials. The format is as follows:
-
-   ```file
-   DATABASE_URL="postgres://__USER__:__PASSWORD__@__HOST__/__DATABASE__"
-   ```
+However, if you use `serverless-webpack`, then you cannot use this method. See [Deployment with serverless webpack](#deployment-with-serverless-webpack) for details.
```

Comment:
Really can not? Or is the plugin just more convenient?

Reply:
closing the loop here: correct, you cannot use this method.